Sources/MongoDriver/MongoDriver.swift

```diff
@@ -68,21 +68,29 @@ extension MongoKitten.Database : Fluent.Driver, Connection {
     ///
     /// TODO: Support all MongoDB operations
     private func makeQuery(_ filters: [RawOr<Filter>], method: Method) throws -> MKQuery {
+
```

Comment:
These whitespace changes should go back to the original code style so it's uniform.